index.d.ts

```diff
@@ -0,0 +1,46 @@
+declare module 'react-form-validator-core' {
+  import * as React from 'react'
+
+  interface ValidatorComponentProps
+    extends React.AllHTMLAttributes<HTMLElement> {
+    errorMessages?: string[] | string
+    validators?: string[]
+    validatorListener?: () => void
+    withRequiredValidator?: boolean
+    onChange: (event: any) => void
+  }
+
+  interface ValidatorComponentState {
+    isValid
```

Comment:
Should it be `boolean`?

Reply:
What do you mean?

Reply:
I've never used typescript, but in other places you have specified type, like
````javascript
input: any
isValid: boolean
````
should be here also
````javascript
interface ValidatorComponentState {
    isValid: boolean
}
````
?

Reply:
O yeah, I missed that one. I will change it.